src/System.Private.ServiceModel/src/System/IdentityModel/Claims/X509CertificateClaimSet.cs

```diff
@@ -267,6 +268,32 @@ public override IEnumerable<Claim> FindClaims(string claimType, string right)
             }
         }
 
+        private static string[] GetDnsFromExtensions(X509Certificate2 cert)
+        {
+            foreach (X509Extension ext in cert.Extensions)
+            {
+                // Extension is SAN or SAN2
+                if (ext.Oid.Value == "2.5.29.7" || ext.Oid.Value == "2.5.29.17")
```

Comment:
[OID-info](http://www.oid-info.com/get/2.5.29.7) says that 2.5.29.7 was declared Obsolete in/prior-to the year 2000; you can probably safely ignore it.  I don't think that Unix builds would even recognize it as anything to pretty print, instead just dumping the bytes in hex.  And certainly SslStream won't use that one